packages/rich-text-editor/src/_mixins.scss

```diff
@@ -32,6 +33,11 @@
     }
   }
 
+  a {
+    color: $color-blue-500;
+    text-decoration: none;
```

Comment:
We may want to provide a hover state - I believe our default is just to add an underline on hover based off the Figma example [here](https://cultureamp.design/guidelines/link-vs-button/) - just confirming with @jmitsios

Reply:
Updated according Jason's feedback. Could I please have another reviews? cc @mcwinter07 Thanks!